# GetPoseResponse.srv
geometry_msgs/Pose pose
bool success

